tests/test_pretty_visualize.py

```diff
@@ -0,0 +1,258 @@
+# (C) Copyright 2018- ECMWF.
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+# In applying this licence, ECMWF does not waive the privileges and immunities
+# granted to it by virtue of its status as an intergovernmental organisation
+# nor does it submit to any jurisdiction.
+
+import re
+from os import remove
+from pathlib import Path
+import pytest
+from conftest import graphviz_present
+from loki import Sourcefile
+from loki.visitors.pretty_visualize import pretty_visualize, Visualizer
+
+
+@pytest.fixture(scope="module", name="here")
+def fixture_here():
+    return Path(__file__).parent
+
+
+test_files = [
+    "trivial_fortran_files/case_statement_subroutine.f90",
```

Comment:
Can we move the entire `trivial_fortran_files` into `tests/sources`?

Reply:
sure no problem a33c38f07e79acb4246f0e705fbd85e9628c31b6